tests/test_global.py

```diff
@@ -0,0 +1,292 @@
+# Run this before deployments, with pytest!
+#
+# The following functionalities are tested:
+# 1. Find download links for certain regions.
+# 2. Cropping satellite images.
+# 3. Adding new bands: NDVI, re_ndvi, NDWI.
+# 4. Cloud detection.
+
+
+import satellite_images_nso.api.nso_georegion as nso
+from settings import nso_username, nso_password, path_geojson, output_path
+import os
+
+# Optional
```

Comment:
If some of the tests here are optional, why not make them into a completely separate file and make that file optional to run?

Reply:
not optional anymore

Reply:
Then perhaps you wish to remove the 'optional' comment?

Reply:
done